src/GifPickerReact.tsx

```diff
@@ -27,7 +29,10 @@ export interface GifPickerReactProps {
 
 function GifPickerReact(props: GifPickerReactProps): JSX.Element {
 	const settings = useSettings(props);
-	const pickerContext = usePickerContext();
+	const pickerContext = usePickerContext({
+		searchTerm: props.searchTerm,
+		setSearchTerm: props.setSearchTerm
+	});
```

Comment:
We are still exposing picker context, just hiding it behind the interface. As  I said I would much rater have one property like `initialSearchTerm`

Reply:
done